spec/openapi.yaml

```diff
@@ -5906,6 +5906,9 @@ components:
       title: summarysubitemresponse
       type: object
       properties:
+        date:
```

Comment:
Can we please add a description:
`One or more dates separated by a comma, e.g. 2022-05-18,2022-05-19`

Reply:
Added!